.github/workflows/rolling-release.yml

```diff
@@ -3,26 +3,30 @@ name: Create rolling release
 on:
   pull_request:
     paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
       - '**.md'
-      - 'appveyor.yml'
+      - '*.yml'
       - 'scripts/*'
-      - '.github/ISSUE_TEMPLATE/*'
   push:
     branches:
       - master
       - dev
     paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
       - '**.md'
-      - 'appveyor.yml'
+      - '*.yml'
       - 'scripts/*'
-      - '.github/ISSUE_TEMPLATE/*'
   workflow_dispatch:
 
 jobs:
-  windows-build:
+  build-windows:
```

Comment:
Renaming all the jobs from `*-build` to `build-*` seems oddly nitpicky to me. It also looks a bit less clear when looking at the jobs list in a workflow run since everything is prefixed with `build`...

Reply:
"windows build" rather than "build windows" makes more sense when reading to me.